### DISCUSSION TITLE
VcOverlayEcharts的使用

### DISCUSSION THREAD

**Discussion by l406054531** [zouyaoji/vue-cesium#411] [Q&A]
1.想点击散点图的时候触发点击事件，但发现文档没有提供点击事件，ref获取VcOverlayEcharts的实例也没有发现有chart的实例
2.我通过定义customClass用echarts的getInstanceByDom获取到chart的实例，然后  chart.on('click',()=>{
    console.log(11);
  })，发现也没有
不知道有没有什么好的解决方法啊

**Comment by zouyaoji** [comment]
请参考这儿的答案

https://github.com/zouyaoji/vue-cesium/discussions/171
